lib/jekyll-admin/directory.rb

```diff
@@ -0,0 +1,62 @@
+module JekyllAdmin
+  class Directory
+    attr_reader :path
+
+    include Enumerable
+    include JekyllAdmin::URLable
+    include JekyllAdmin::APIable
+
+    TYPE = 'directory'
+
+    def initialize(path, args={})
+      @args = args
+      @base = Pathname.new args[:base]
+      @content_type = args[:content_type]
+      @splat = Pathname.new args[:splat]
+      @path = Pathname.new path
+    end
+
+    def to_liquid
+      {
+        name: name,
+        modified_time: modified_time,
+        path: relative_path,
+        type: TYPE
+      }
+    end
+
+    def name
+      @path.basename
```

Comment:
1. Since you have `attr_reader :path` above, this can be just `path.basename`

2. To DRY things up a bit, you can add the following in place of the `name` and `modified_time` methods:

```ruby
  extend Forwardable
  def_delegator :@path, :basename, :name
  def_delegator :@path, :mtime, :modified_time
```